src/app/components/apparatus-entry/apparatus-entry.component.html

```diff
@@ -0,0 +1,3 @@
+<span class="evt-app-entry" [attr.data-variance]="variance$ | async">
+   <evt-content-viewer *ngIf="data.lemma" [content]="data.lemma" [editionLevel]="editionLevel"></evt-content-viewer>
```

Comment:
I would add the bind to `itemsToHighlight`, too.

Reply:
Isn’t it better to add it to the reading component?